lib/elixir_google_scraper/scraper.ex

```diff
@@ -9,6 +9,16 @@ defmodule ElixirGoogleScraper.Scraper do
   alias ElixirGoogleScraper.Repo
   alias ElixirGoogleScraper.Scraper.{CSVKeyword, Keyword}
 
+  def paginated_user_keywords(user, params \\ %{}) do
+    user
+    |> user_keywords_query
+    |> Repo.paginate(params)
+  end
+
+  defp user_keywords_query(user) do
```

Comment:
Let's move this function to the bottom of the file to group private methods together.